src/librustdoc/html/render/span_map.rs

```diff
@@ -46,17 +59,26 @@ pub(crate) fn collect_spans_and_sources(
     src_root: &Path,
     include_sources: bool,
     generate_link_to_definition: bool,
-) -> (FxIndexMap<PathBuf, String>, FxHashMap<Span, LinkFromSrc>) {
+    generate_macro_expansion: bool,
+) -> (
+    FxIndexMap<PathBuf, String>,
+    FxHashMap<Span, LinkFromSrc>,
+    FxHashMap<BytePos, Vec<ExpandedCode>>,
```

Comment:
The doc comment says:

> It returns the `krate`, the source code files and the `span` correspondence map.

The return value is changed, so the docs should be, too.

Reply:
The doc was actually already outdated, fixed both cases.